pkg/utils/validation/features/featuregates.go

```diff
@@ -138,7 +138,7 @@ var featureGateVersionRanges = map[string]*FeatureGateVersionRange{
 	"IPTablesOwnershipCleanup":                         {LockedValue: true, LockedToDefaultInVersion: "1.28", VersionRange: versionutils.VersionRange{RemovedInVersion: "1.30"}},
 	"ImageMaximumGCAge":                                {VersionRange: versionutils.VersionRange{AddedInVersion: "1.29"}},
 	"ImageVolume":                                      {VersionRange: versionutils.VersionRange{AddedInVersion: "1.31"}},
-	"InPlacePodVerticalScaling":                        {},
+	"InPlacePodVerticalScaling":                        {LockedValue: true, VersionRange: versionutils.VersionRange{AddedInVersion: "1.27"}},
```

Comment:
`LockedValue` is only used if the feature gate is also locked to default. In k8s 1.33 the feature gate was only changed to be enabled by default, but it was not also locked to its default value
```suggestion
	"InPlacePodVerticalScaling":                        {VersionRange: versionutils.VersionRange{AddedInVersion: "1.27"}},
```

Reply:
Thanks for the suggestion, removed the `LockedValue` property in my latest updates to keep the `InPlacePodValidationScaling` gate consistent with the rest of the entries. 

From what I saw

https://github.com/gardener/gardener/commit/2c4f3e96b2c#diff-81136865f4bd7a1313a9b25446c6abc89f76cc72b01e6000f1c71f97ce620683R333-R335

changes the `Default` to `LockedValue`, however it seems that it's behaviour is preserved, as the

https://github.com/gardener/gardener/blob/2c677126795dd8587acdec9f2e8fc651ccde7160/pkg/utils/validation/features/featuregates.go#L338-L348

utility used to determine the `locked` state of a gate does not make use of it.

Nonetheless, it's better to keep it consistent with the rest of the gates, hence I apply the suggestion :)) Thanks!